content/blue_green_deployments/tabs/aws-console.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Embedded tab content"
+disableToc: true
+hidden: true
+---
+
+#### Navigate to the LoadBalancer endpoint
+You can use the output value of `BlueGreenUsingEcsStack.ecsBlueGreenLBDns`. This is deployed using a demo docker image from docker registry. We will now modify this deployment using the CodePipeline built by the CDK stack in the previous section.
+
+![blue-deployment](/images/blue-deployment.png)
+
+#### Navigate to the AWS console
+
+* When you navigate to the AWS console, you will see a CodePipeline with a failure. I am using `us-east-1 - N. Virginia` region. Choose the region where you had deployed the CDK stack
+
+![aws-console-1](/images/blue-green-aws-console-1.png)
+
+* Click on the pipeline and you can see this is due to missing source code in the CodeCommit repository
```

Comment:
I don’t see a lot of value in seeing the failed pipeline, this could feel like something is broken and this is a workaround of some sort. Let’s skip showing the failed pipeline and go straight to the git clone/edit/push process. Once we push the code up, we can then go to the pipeline and see it working in realtime.

Reply:
The workflow i'm hoping for is something like this: 

- deploy the environment via cdk
- look at ALB URL (shows blue)
- go to command line and update nginx/push to git
- navigate to pipeline, walkthrough what pipeline is doing
- dive into blue green deployment section (codedeploy). Walk through what it's doing and what each step represents. Also some mention around rollbacks and the options available during the deploy

Reply:
Removed the failed pipeline section and updated the workflow